[INFO -- 2018-11-23 18:47:56]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-23 18:47:56]
Initialization strings:
[INFO -- 2018-11-23 18:47:56]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-23 18:47:56]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-11-23 18:47:56]
initSignal:0
[INFO -- 2018-11-23 18:47:56]
endSignal:3
[INFO -- 2018-11-23 18:47:56]
faultModel:2
[EXCEPTION -- 2018-11-23 18:47:57]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-11-23 18:47:57]
gdbException: Cannot access memory at address 0x15350d5bb9517d4a
[INFO -- 2018-11-23 18:47:57]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec6801 in __GI_abort () at abort.c:79
#2  0x00007ffff7ad88b7 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7adea06 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7adea41 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7adec74 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff7b06ed2 in std::__throw_bad_alloc() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#7  0x00005555555577e4 in __gnu_cxx::new_allocator<double>::allocate (this=0x7fffffffd8a0, __n=18255738009813299905) at /usr/include/c++/7/ext/new_allocator.h:102
#8  0x000055555555757d in std::allocator_traits<std::allocator<double> >::allocate (__a=..., __n=18255738009813299905) at /usr/include/c++/7/bits/alloc_traits.h:436
#9  0x0000555555557054 in std::_Vector_base<double, std::allocator<double> >::_M_allocate (this=0x7fffffffd8a0, __n=18255738009813299905) at /usr/include/c++/7/bits/stl_vector.h:172
#10 0x0000555555556db9 in std::_Vector_base<double, std::allocator<double> >::_M_create_storage (this=0x7fffffffd8a0, __n=18255738009813299905) at /usr/include/c++/7/bits/stl_vector.h:187
#11 0x0000555555556489 in std::_Vector_base<double, std::allocator<double> >::_Vector_base (this=0x7fffffffd8a0, __n=18255738009813299905, __a=...) at /usr/include/c++/7/bits/stl_vector.h:138
#12 0x0000555555555f35 in std::vector<double, std::allocator<double> >::vector (this=0x7fffffffd8a0, __x=std::vector of length -191006063896251710, capacity -191006063896251710 = {...}) at /usr/include/c++/7/bits/stl_vector.h:328
#13 0x00005555555556c3 in train (iterations=20000, alpha=5.0000000000000002e-05, N=3, stopsWhenStable=true) at lreg.cpp:126
#14 0x0000555555555ab7 in main (argc=3, argv=0x7fffffffde58) at lreg.cpp:174

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Memory content before bitflip:0x060x000x000x00
Fault Model: Random bit-flip
base address to flip value: 0x7fffffffd92c
address max offset: 4
Memory content after  bitflip:0x580x260xd40xf4
frame name: main
symbol name: predictions
symbol filename: lreg.cpp
symbol line: 161
value: -187423144
value address: 0x7fffffffd92c
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2018-11-23 18:47:57]
Fault Injection Successful after 1.4671437740325928s
[INFO -- 2018-11-23 18:47:57]
event type: exit
[EXCEPTION -- 2018-11-23 18:47:57]
exit code: no exit code available
